math/mathcore/inc/TMath.h

```diff
@@ -271,6 +285,20 @@ constexpr Double_t KUncertainty()
    return 0.0;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Coulomb Constant in \f$ kgm^3s^{-2}C^{-2} \f$
+constexpr Ke()
```

Comment:
Please add here the declaration (e.g. Double_t)

Reply:
Ah, ok.